site/docs/webrtc/voice-iw.md

```diff
@@ -0,0 +1,191 @@
+
+```js
+    const callSipUri = async (participant: ParticipantInfo) => {
+        const body = {
+            from: voiceApplicationPhoneNumber,
+            to: "sip:sipx.webrtc.bandwidth.com:5060",
+            answerUrl: `${voiceCallbackUrl}/bridgeCallAnswered`,
+            uui: `${participant.token};encoding=jwt`,
+```
+
+This [/bridgeCallAnswered](<https://github.com/Bandwidth-Samples/webrtc-voicebridge-ts/blob/main/src/server.ts#:~:text=*/-,app.post(%22/bridgeCallAnswered%22%2C%20async%20(req%2C%20res)%20%3D%3E%20%7B,%7D)%3B,-/**>) API call links your WebRTC session with Programmable Voice. Then, Bandwidth's Voice API will hit the [/callAnswered](<https://github.com/Bandwidth-Samples/webrtc-voicebridge-ts/blob/main/src/server.ts#:~:text=app.post(%22/callAnswered,%7D)%3B>) endpoint when an outgoing call is answered. The two independent calls can be established synchronously or in parallel, providing the flexibility in management of the endpoints. For example, a single webRTC-side call can be established that connects to a series of PV-side calls, without the need to set up the entire end-end experience for each new voice caller. You can find the complete code and instructions for the sample above in the [bridge repository](https://github.com/Bandwidth-Samples/webrtc-voicebridge-ts).
+
+If reducing delays in interconnection of voice and WebRTC legs is important, the best option would be to use the bridge verb, which allows setting up the legs in parallel, or establishing one leg early in anticipation of the end to end call.
+
+## Conference
+
+Although use of the Conference verbe has higher costs than the previous methods, it would be the most sustainable option for dealing with a large group of voice callers. If you expect more than two voice callers in one session, the Conference method is effective if Programmable Voice services are needed for those callers. You can have 20 calls in a conference and keep the session for 24 hours. You also have the freedom to add and remove participants from the conference.
+
+![Conference Verb Model](../../static/img/webrtc-pv-conference.png "Conference")
+
+Conferencing is actively supported by both the WebRTC and Voice platforms, however, if both platforms are interconnected, and the power of PV is needed on the voice-only call legs, use of the Conference Bxml verb has value. If fine-grained control over media in a conference scenario is needed, or of video is required in the conference, it is likely wise to use the conferencing capability inherent in the WebRTC platform and APIs.
+
+Here’s a [code snippet](https://github.com/Bandwidth-Samples/webrtc-voiceconf-ts) of how the Conference verb is used:
+
+Bandwidth's Voice API calls the webRTC infrastructure with the participant token in the UUI SIP header to allow the correlation ofV2 voice and the webRTC infrastructure
```

Comment:
webRTC

Reply:
No space between of and V2

Reply:
You might want for format UUI as code throughout

Reply:
I fixed the middle bug, but I'm not sure I understood the first and 3rd comment.  I meddled with the wording anyway :-)

Reply:
got it - also resolved a couple of other webRTC instances in the document text.